go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	github.com/onflow/flow-core-contracts/lib/go/contracts v1.6.0
 	github.com/onflow/flow-core-contracts/lib/go/templates v1.6.0
 	github.com/onflow/flow-go-sdk v1.3.3
-	github.com/onflow/flow/protobuf/go/flow v0.4.9
+	github.com/onflow/flow/protobuf/go/flow v0.4.10-0.20250226225747-321bed061708
```

Comment:
TODO: Merge https://github.com/onflow/flow/pull/1558, tag a version, and update here.

Reply:
https://github.com/onflow/flow-go/pull/6955/commits/655da025a0ef0155d4c9ceddcc2dae55eee309bb